package.json

```diff
@@ -6,14 +6,18 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "cypress run",
```

Comment:
Great, this is nice to see the setup for cypress run early and ready before testing commences ✨ 🧪